src/lib/card/card.scss

```diff
@@ -70,8 +74,8 @@ $mat-card-header-size: 40px !default;
 }
 
 .mat-card-image {
-  width: calc(100% + 48px);
-  margin: 0 -24px 16px -24px;
+  width: calc(100% + #{$mat-card-padding * 2});
+  margin: 0 $mat-card-padding * -1 16px $mat-card-padding * -1;
```

Comment:
Out of curiosity: why not just `-$mat-card-padding`? Maybe just a preference though.

Reply:
It's because SASS evaluates the last two margins to `16px - $mat-card-padding`.

Reply:
Oh I see. Thanks